src/components/RichText/RichTextField.tsx

```diff
@@ -132,6 +137,20 @@ export function RichTextField({
   );
 
   const val = input.value as RichTextData | undefined;
+
+  useEffect(() => {
```

Comment:
We could use the `useEventListener` helper from `ahooks`. We have a ref to that element as well so no need to query the global document for it.
```ts
useEventListener('paste', (e: ClipboardEvent) => {
  ...
}, { target: ref });
```